src/lando/pushlog/models/commit.py

```diff
@@ -239,13 +239,52 @@ class Tag(models.Model):
         on_delete=models.SET_NULL,
     )
 
+    _scm_commit: CommitData | None = None
+
+    @staticmethod
```

Comment:
Looks like this should be `@classmethod` and the use `cls` instead of `Tag` below?